transfer.md

```diff
@@ -1,15 +1,22 @@
 # CRI-O Usage Transfer
 
-This document outlines useful information for operations and development transfer as it relates to infrastructure that utilizes CRI-O.
+This document outlines useful information for operations and development transfer
+as it relates to infrastructure that utilizes CRI-O.
 
 ## Operational Transfer
 
 ## Abstract
 
-The `crio` daemon is intended to provide the CRI socket needed for Kubernetes to use for automating deployment, scaling, and management of containerized applications (See the document for [configuring kubernetes to use CRI-O](./tutorials/kubernetes.md) for more information).
-Therefore the [crictl][1] command line is a client that interfaces to the same gRPC socket as the kubernetes daemon would, for talking to the `crio` daemon.
+The `crio` daemon is intended to provide the CRI socket needed for Kubernetes to
```

Comment:
out of scope for here: opened https://github.com/cri-o/cri-o/issues/6896 because a lot of this info isn't right

Reply:
Do you want to keep it like this or revert the changes and ignore the file until #6896

For now I will add this notice:

:warning: **Attention**: This documentation is out of date and requires updating. Please review and revise accordingly. #6896

Reply:
> Do you want to keep it like this or revert the changes and ignore the file until #6896

I say let's just ignore the file

Reply:
I reverted my changes and ignored the file with a comment. The warning can't hurt i guess.